crates/rooch-types/src/block.rs

```diff
@@ -11,32 +15,64 @@ pub struct Block {
     pub block_number: u128,
     /// How many transactions in the block
     pub batch_size: u64,
-    /// The hash of the batch, made by DA
-    pub batch_hash: H256,
+    /// The hash of the block, made by DA
+    pub block_hash: H256,
```

Comment:
I used batch here, because DA only care about tx list. Not state made by present. If we call it block hash, may hint there is execution inside it? Actually, this hash contains no result of tx execution at all.

Which proposer needs is a hash which combine state and tx itself(state_root & batch_hash)

Reply:
Yes, the block hash in the proposer needs to be associated with the transaction (need to check the block based on tx hash or tx order). This is somewhat different from the batch hash in DA, but it would be best if they could be unified.